instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/semconv/http/HttpClientAttributesExtractor.java

```diff
@@ -32,6 +35,9 @@ public final class HttpClientAttributesExtractor<REQUEST, RESPONSE>
         REQUEST, RESPONSE, HttpClientAttributesGetter<REQUEST, RESPONSE>>
     implements SpanKeyProvider {
 
+  private static final Set<String> PARAMS_TO_REDACT =
```

Comment:
Is it necessary to provide an environment variable to the user so that the user can set some keys themselves?

Reply:
Make the parameter name configurable is not in the scope of the specifications today: https://github.com/open-telemetry/semantic-conventions/blob/main/docs/http/http-spans.md

I would prefer to have a first PR on the current specifications, and potentially other ones to go beyond the current specifications.